### ISSUE TITLE
keras.layers.merge.Add() cannot work

### ISSUE THREAD

**Issue by haruji** [keras-team/keras#7969]
It seems that keras.layers.merge.Add()[x,y] cannot work correctly.
The error information is ' AttributeError:'function'object has no attribute 'Add' '

If I erase ‘merge’, using keras.layers.Add()[x,y], the program can run properly.

But according to the docs, the Add layer is in the merge layer. 
So in my opinion, either way should be executable? Is something wrong?

**Comment by facaiy** [comment]
I suggest to use `keras.layers.Add`. As far as I known, all ops under layers are exposed at `keras.layers`, for more to see https://github.com/fchollet/keras/blob/710898f759b9ec229f3513bb9f58fd032c1aaed6/keras/layers/__init__.py#L8

As for the case, unfortunately, `merge` method (legacy) masks the module. https://github.com/fchollet/keras/blob/710898f759b9ec229f3513bb9f58fd032c1aaed6/keras/legacy/layers.py#L399
